app/inpututils.cpp

```diff
@@ -286,8 +286,13 @@ bool InputUtils::cpDir( const QString &srcPath, const QString &dstPath )
     {
       if ( !QFile::copy( srcItemPath, dstItemPath ) )
       {
-        qDebug() << "Cannot copy file " << srcItemPath << " > " << dstItemPath;
-        return false;
+        QFile::remove( dstItemPath );
+        if ( !QFile::copy( srcItemPath, dstItemPath ) )
+        {
+          qDebug() << "Cannot copy file " << srcItemPath << " > " << dstItemPath;
+          return false;
```

Comment:
how about we do not return immediately and just skip the file? So we will make the best effort to copy all the files...